apps/backend/src/datasources/postgres/InstrumentDataSource.ts

```diff
@@ -667,6 +667,48 @@ export default class PostgresInstrumentDataSource
     );
   }
 
+  async unsubmitInstrument(
+    proposalPks: number[],
+    instrumentId: number
+  ): Promise<InstrumentsHasProposals> {
+    const records: FapProposalRecord[] = await database('fap_proposals')
+      .update(
+        {
+          fap_meeting_instrument_submitted: false,
+        },
+        ['*']
+      )
+      .whereIn('proposal_pk', proposalPks)
+      .andWhere('instrument_id', instrumentId);
+
+    if (!records?.length) {
+      throw new GraphQLError(
+        `Some record from fap_proposals not found with proposalPks: ${proposalPks} and instrumentId: ${instrumentId}`
```

Comment:
Maybe better error  message could be 
`No fap_proposals found with proposalPks: ${proposalPks} and instrumentId: ${instrumentId}`